e2e/core/pages/metadata-view.page.ts

```diff
@@ -311,4 +321,12 @@ export class MetadataViewPage {
     async clickResetMetadata(): Promise<void> {
         await BrowserActions.click(this.resetMetadataButton);
     }
+
+    async clickSaveGeneralMetadata(): Promise<void> {
+        await BrowserActions.click(this.saveGeneralMetadataButton);
+    }
+
+    async generalSaveIconDisplayed(): Promise<void> {
```

Comment:
that name of function seems incorrect - it says displayed but below you have IsNotVisible function used